src/basic/string-util.c

```diff
@@ -645,21 +645,21 @@ char *strip_tab_ansi(char **ibuf, size_t *_isz) {
                         else if (*i == '\x1B')
                                 state = STATE_ESCAPE;
                         else if (*i == '\t')
-                                fputs("        ", f);
+                                fputs_unlocked("        ", f);
```

Comment:
looks excellent, but could you add a brief line about this aftet the open_memstring() line? something along the lines of "please note we use the 'unlocked' version of the stdio calls, as we know we are the only ones accessing the STDIO objects as we just allocated them here, and this does make quite a performance difference"... or so?